gtfs/spec/en/reference.md

```diff
@@ -394,7 +394,7 @@ Methods for occupancy forecasting used to populated `occupancies.txt` are not st
 
 | Field Name | Type | Required | Description |
 | ----- | ----- | ----- | ----- |
-| `trip_id` | ID referencing `stop_times.trip_id` | **Required** |  Identifies a `trip_id` for which an occupancy level is described. |
+| `trip_id` | ID referencing `trips.trip_id` | **Required** |  Identifies a `trip_id` for which an occupancy level is described. |
```

Comment:
Are you sure you want to do a compound key to a different table?

Reply:
I was thinking since `stop_sequence` is optional it would make sense to reference `trips.trip_id` since it can live on its own. But we should reference whatever is less expensive for consumers. So `stop_times.trip_id` is indeed more favorable?

Reply:
I didn't think of the situation where the trip itself had a single occupancy prediction...

Reply:
Technically a trip with only 2 stops could use trip-level occupancy. But I agree, the gain of not including `stop_sequence` is pretty marginal. Are you suggesting me make `stop_sequence` required then, or should it remain optional?

Reply:
From my point of view it is "better" to have a stop_sequence prediction. Are there any producers that can only do trip level predictions but not stop level ones?

Reply:
I agree it is better. I would think trip level predictions are uncommon.

Reply:
Speaking from the US Commuter Rail producer side, trip-level predictions currently **are** the norm (e.g. Boston, Chicago, Los Angeles).

While we'd love to provide more granular stop-by-stop ridership data, most of us don't have robust APC systems, which forces us to rely on conductor counts, manual fare collection data, platform counts at downtown terminals, etc.

Since suburb-to-city ridership still dominates travel patterns of North American legacy systems, we've found that providing trip-level crowding assessed at the maximum load point has been a good proxy for customers, particularly since the concern of most customers is how crowded their train *will be*, rather than what it's like should they board at an early stop.

Yes, this overestimates the impact for customers traveling between stations on the outer edge. Yes, this doesn't discern where along the line the train will fill. Yes, there's far more detail that customers would find of value. But this still allows us to provide significant insights given the limited data we have… and I don't want to be yet another provider to release this data in a bespoke format (or worse, something that's not machine-readable like a PDF).